### PR TITLE
Fix token list on sandpit

### PR DESCRIPTION
**Pull request by iverc**

### CODE REVIEW

File: ui/app/src/components/shared/DetailsPanel.vue [hunk 1]
```diff
@@ -61,10 +61,9 @@ export default defineComponent({
           </Tooltip>
         </span>
 
-        <span
-          >{{ formatNumber(minimumReceived) }}
-          <span>{{ toToken.toUpperCase().replace("C", "c") }}</span></span
-        >
+        <span>{{ formatNumber(minimumReceived) }}
+          <span>{{ toToken.slice(1).toUpperCase() }}</span>
```
Comment (59023g) [inline ui/app/src/components/shared/DetailsPanel.vue:65]: What's this

**Review by 59023g — APPROVED** [review]

File: ui/app/src/components/shared/DetailsPanel.vue (hunk 1 above)
Comment (iverc) [reply ui/app/src/components/shared/DetailsPanel.vue:65]: @59023g if we replace C with c it will mess up tokens like USDc

Comment (59023g) [reply ui/app/src/components/shared/DetailsPanel.vue:65]: but it's `cUSDC`

Comment (59023g) [reply ui/app/src/components/shared/DetailsPanel.vue:65]: Also, `slice` will remove string character at index 0. Please revert this.

Comment (59023g) [reply ui/app/src/components/shared/DetailsPanel.vue:65]: Also, slice will remove string character at index 0. Please revert this.

**Review by 59023g — CHANGES_REQUESTED** [review]

**Review by 59023g — APPROVED** [review]
🆒

**Pull request merged**